Bio/motifs/meme.py

```diff
@@ -144,6 +128,7 @@ def __getitem__(self, key):
 
 
 def __read_version(record, handle):
+    #This function still works
```

Comment:
This doesn't seem like a useful comment line to add.

Reply:
Also in terms of the automatically enforced coding style (using ``flake8``), this should be hash then space:

```
Bio/motifs/meme.py:131:5: E265 block comment should start with '# '
```

But please just remove the comment line completely.

Reply:
I am the most experience Graphic Designer ... 
https://fiverr.com/share/4m9wx